Makefile

```diff
@@ -6,7 +6,13 @@ RS_AST_OUT = rust/ast/src/types_generated.rs \
 	rust/ast/src/visit_generated.rs \
 	rust/ast/src/source_location_accessor_generated.rs \
 	rust/generated_parser/src/stack_value_generated.rs
-PYTHON = python3
+PYTHON = $(shell pwd)/jsparagus_build_venv/bin/python3
```

Comment:
I'd like this to work even if someone does `make -k js/jsparagus/Makefile` from some other directory, so:

```
JSPARAGUS_DIR := $(dir $(firstword $(MAKEFILE_LIST)))
VENV_BIN_DIR := $(JSPARAGUS_DIR)/jsparagus_build_venv/bin
PYTHON := $(VENV_BIN_DIR)/python
PIP := $(VENV_BIN_DIR)/pip
```